build/images/Dockerfile.build.agent.coverage

```diff
@@ -37,10 +37,8 @@ USER root
 
 COPY build/images/scripts/* /usr/local/bin/
 COPY --from=antrea-build /antrea/bin/antrea-agent /usr/local/bin/
```

Comment:
nit: @shikharish Can we rename the binaries as part of the `COPY` statement and remove the `mv` commands above?

For example:
```
COPY --from=antrea-build /antrea/bin/antrea-agent-coverage /usr/local/bin/antrea-agent

```